src/app/create-event/page.tsx

```diff
@@ -26,6 +26,23 @@ export default function CreateEvent() {
 
   const [isButtonsVisible, setIsButtonsVisible] = useState(false) // New state to control visibility of buttons
 
+  useEffect(() => {
+    // Check if user is signed in
+    getUser(localStorage.getItem('username') as UUID)
```

Comment:
Shouldn't there be a username check here to see if local storage has this before using it?

Reply:
That is to prevent an error message when it isn't a real error.